examples/ASMapNode/Sample.xcodeproj/project.pbxproj

```diff
@@ -178,6 +181,21 @@
 /* End PBXResourcesBuildPhase section */
 
 /* Begin PBXShellScriptBuildPhase section */
+		0518F692ABB8E763D9C401ED /* 📦 Embed Pods Frameworks */ = {
```

Comment:
It seems like you are using an older version of Cocoapods as it has emojis in there. Now it's like `[CP]` or something like that. Can you please  revert the changes with emojis in there? Thanks!